CONTRIBUTING.rst

```diff
@@ -237,7 +237,7 @@ There are a few coding conventions we use in beets:
    .. code-block:: python
 
        with g.lib.transaction() as tx:
-             rows = tx.query('SELECT DISTINCT "{0}" FROM "{1}" ORDER BY "{2}"'
+             rows = tx.query("SELECT DISTINCT '{0}'' FROM '{1}'' ORDER BY '{2}'""
```

Comment:
I think this needs fixing

Reply:
I should not try and code after an afternoon at the football 😃